### PR TITLE
Example app update

### PR DESCRIPTION
**Pull request by croossin**
This is in response to issue #7 .  Will give more examples about how all different CardParts work.

### CODE REVIEW

**Comment by lwdupont** [comment]
Pulling it and checking it out now. Thanks!!

**Comment by croossin** [comment]
@lwdupont I definitely want to add another tab in regards to the [different traits](https://github.com/intuit/CardParts#card-traits).  Or some more complex examples. But this certainly gives a lot more preview of what CardParts can do.

File: Example/CardParts/CardPartBarViewCardController.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//
+//  CardPartBarViewCardController.swift
+//  CardParts_Example
+//
+//  Created by Roossin, Chase on 5/23/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
```
Comment (lwdupont) [inline Example/CardParts/CardPartBarViewCardController.swift]: Did you mean to have these marked CocoaPods?

Comment (bharathmurs) [reply Example/CardParts/CardPartBarViewCardController.swift]: Oops :D

File: Example/CardParts/CardPartButtonViewCardController.swift [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    
+    let cardPartTextView = CardPartTextView(type: .normal)
+    let cardPartButtonView = CardPartButtonView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        cardPartTextView.text = "This is a CardPartButtonView"
+        
+        cardPartButtonView.setTitle("I'm a button!", for: .normal)
```
Comment (bharathmurs) [inline Example/CardParts/CardPartButtonViewCardController.swift]: Can you also add target block? to show how button taps are handled?

File: Example/Podfile.lock [hunk 3]
```diff
@@ -1,5 +1,5 @@
 PODS:
```
Comment (bharathmurs) [inline Example/Podfile.lock]: Probably gitignore this? 
and pod install every time.

**Pull request merged**